rental.rb

```diff
@@ -1,37 +1,23 @@
 class Rental
   attr_accessor :date, :book, :person
 
+  @rentals = [] # Class instance variable to store all rentals
```

Comment:
Great job @elad237 🤝

- [ ] [Optional] Since the previous reviewer could not run the app to test all use cases, he could not create a rental and also list rentals for a person. Kindly ensure your app can perform these actions. When I try to add a rental I get this error. Kindly fix it to make your project complete.
![rental](https://github.com/elad237/OOP-school-library/assets/63917725/e233534f-1361-45f9-bbd1-9ed75adcd663)